File: aoj/volume100/java/A10007.java
import java.util.Scanner;

class A10007 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        while (true) {
            int a = scan.nextInt();
            int b = scan.nextInt();
            if (a == 0 && b == 0)
                break;

            System.out.printf("%d %d\n", Math.min(a, b), Math.max(a, b));
        }
    }
}

File: aoj/volume100/ocaml/A10007.ml
let rec loop () =
    let (a, b) = Scanf.scanf "%d %d\n" (fun a b -> (a, b)) in
    match (a, b) with
    | (0, 0) -> ()
    | (_, _) -> (
        Printf.printf "%d %d\n" (min a b) (max a b) ;
        loop () ;
    )
in
loop ()
